core/server/data/meta/description.js

```diff
@@ -14,7 +14,7 @@ function getDescription(data, root) {
     } else if (_.contains(context, 'author') && data.author) {
         description = data.author.bio;
```

Comment:
I was only copying what was currently being used currently. If you look at the meta_description helper  it wasn't using meta_description but I can make the change tonight.